api/deploy_deployer.go

```diff
@@ -105,10 +105,27 @@ func (d *serviceDeployer) deploy(path string) (*service.Service, *importer.Valid
 		d.sendStatus(fmt.Sprintf("%s [DEPRECATED] Please use .dockerignore instead of .mesgignore", aurora.Red("⨯")), DONE)
 	}
 	d.sendStatus(fmt.Sprintf("%s Image built with success.", aurora.Green("✔")), DONE)
-	d.injectConfigurationInDependencies(service, imageHash)
+	d.adjustFields(s)
+	d.injectConfigurationInDependencies(s, imageHash)
 
 	d.sendStatus(fmt.Sprintf("%s Completed.", aurora.Green("✔")), DONE)
-	return service, nil, services.Save(service)
+	return s, nil, services.Save(s)
+}
+
+func (d *serviceDeployer) adjustFields(s *service.Service) {
```

Comment:
No comment on method + adjustFileds tells nothing about what function does.

Reply:
i'll remove that func within the next or 2nd PR

Reply:
kk